00-introduction.md

```diff
@@ -21,6 +21,7 @@ This is version 0.
 7. [BOLT #7](07-routing-gossip.md): P2P Node and Channel Discovery
 8. [BOLT #8](08-transport.md): Encrypted and Authenticated Transport
 9. [BOLT #9](09-features.md): Assigned Feature Flags
+11. [BOLT #11](11-payment-encoding.md): Invoice Protocol for Lightning Payments
```

Comment:
BIPs have never been limited to Core or Bitcoin consensus. They encompass anything within the scope of Bitcoin that needs standardisation. BU trolls may as part of their agenda to hijack Bitcoin, try to spin BIPs as limited in scope to a specific part of Bitcoin, but this is simply not true.